### ISSUE TITLE
Fine tuning on subset of ADE20K dataset for segmentation

### ISSUE THREAD

**Issue by amil-rp-work** [dmlc/gluon-cv#1437]
Hey all
I have tried to leverage the pre-trained models for semantic segmentation, however the results are not great for my use-case. I was wondering if it's possible to fine-tune the segmentation models on a subset of classes of ADE20k dataset. I was unable to find some helper function or info about the labels in each image. 
Any help will be appreciated thanks!

**Comment by bryanyzhu** [comment]
Hi, for information about labels in each image, you can see our tutorial for how to prepare ADE20K dataset (https://gluon-cv.mxnet.io/build/examples_datasets/ade20k.html), and the class names in (here)[https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/ade20k/segmentation.py#L38-L98]

If you want to train on a subset of classes in ADE20K dataset, just select the classes you want in that big list and delete the rest. And change `NUM_CLASS` accordingly (https://github.com/dmlc/gluon-cv/blob/master/gluoncv/data/ade20k/segmentation.py#L37). The rest should stay the same, and you can train a segmentation model on your dataset. Hope this helps, let me know if you have additional questions.

**Comment by amil-rp-work** [comment]
@bryanyzhu Sorry for the late reply, seems to work well. Thanks a lot for the help !!!
